doc/source/_static/css/custom.css

```diff
@@ -151,3 +151,14 @@ img.horizontal-scroll {
 pre {
     white-space: pre-wrap;
 }
+
+/* notebook formatting */
+.cell .cell_output {
+    max-height: 250px;
+    overflow-y: auto;
```

Comment:
@krfricke I'm not sure we always want this behaviour, but let's go for it for now. Won't hurt much at this point